components/tile/_tile.scss

```diff
@@ -135,10 +141,9 @@
 		width: $tile-width-less-padding;
 		height: $footer-height;
 		@include box-sizing;
-
-		border-top: 1px solid $tlv-light-gray;
+		border-top: 1px solid $light-gray;
 		margin-top: 7px;
```

Comment:
remove margin-top, if the text above the footer need gap from the line, add padding-bottom to sdc-tile-content